numba_dppy/dppy_parfor_diagnostics.py

```diff
@@ -0,0 +1,104 @@
+from numba.parfors.parfor import ParforDiagnostics, _termwidth, print_wrapped
+
+
+class ExtendedParforDiagnostics(ParforDiagnostics):
+    def __init__(self):
+        ParforDiagnostics.__init__(self)
+        self.extra_info = {}
+
+    def dump(self, level=1):
+        ParforDiagnostics.dump(self, level=1)
+
+        if self.extra_info:
+            parfors_simple = self.get_parfors_simple(False)
+            all_lines = self.get_all_lines(parfors_simple)
+            print(' Auto-offloading '.center(_termwidth,'-'))
+            self.print_auto_offloading(all_lines)
+            if 'kernel' in self.extra_info.keys():
+                print_wrapped("Device - '%s'" % self.extra_info['kernel'])
+            print(_termwidth * '-')
+
+    def print_auto_offloading(self, lines):
```

Comment:
Are we overriding these methods? Is it to have more control on what we are printing?